### PR TITLE
Feature/sergeyro/chips elipsis tooltip

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/3868111735

### CODE REVIEW

File: src/components/Chips/Chips.tsx
Comment (SergeyRoyt) [inline src/components/Chips/Chips.tsx:208]: in return statement changes only here

File: src/components/Chips/Chips.tsx
Comment (SergeyRoyt) [inline src/components/Chips/Chips.tsx:230]: and here

File: src/components/Chips/Chips.tsx
Comment (hadasfa) [inline src/components/Chips/Chips.tsx:27]: we need to remember update the tooltip pr after this merged

File: src/components/Chips/Chips.tsx
Comment (hadasfa) [inline src/components/Chips/Chips.tsx:194]: did not get this change, if it's not overflowing we want not aria label? What the aria label is affected by overflow and not always ariaLabel || label

File: src/components/Chips/Chips.tsx
Comment (hadasfa) [reply src/components/Chips/Chips.tsx:208]: did not understand this note?

File: src/components/Chips/Chips.tsx
Comment (hadasfa) [inline src/components/Chips/Chips.tsx:193]: I think the isOverflowing logic make the code more complicated and not organized (there is logic here for tooltip and for focus which depend on this).
I think it will be more organized if we will create a hook in the chips folder which called: useOverflowTooltip which return wrapperProps and tooltipProps.

File: src/components/Chips/__stories__/chips.stories.mdx
Comment (hadasfa) [inline src/components/Chips/__stories__/chips.stories.mdx:155]: Is it according to design review?

Comment (SergeyRoyt) [reply src/components/Chips/__stories__/chips.stories.mdx:155]: It's just an "Demonstration examples" :) Probably don't want to show it at all in storybook - I've asked Meirav about that, waiting for an answer

Comment (SergeyRoyt) [reply src/components/Chips/__stories__/chips.stories.mdx]: Removed it for now

File: src/components/Chips/Chips.tsx
Comment (SergeyRoyt) [reply src/components/Chips/Chips.tsx]: Did it, thanks!

File: src/components/Chips/Chips.tsx
Comment (SergeyRoyt) [reply src/components/Chips/Chips.tsx]: Fixed

File: src/components/Chips/Chips.tsx
Comment (SergeyRoyt) [reply src/components/Chips/Chips.tsx]: Nothing bad will happen even if we forget to

Comment (hadasfa) [reply src/components/Chips/Chips.tsx]: i guess but there will no need in the Tooltip.hideShowTriggers.FOCUS so we can remove it there

File: src/components/Chips/Chips.tsx
Comment (hadasfa) [inline src/components/Chips/Chips.tsx:115]: a bit confusing, why it's not: 
ariaLabel || label;

File: src/components/Chips/Chips.tsx
Comment (hadasfa) [inline src/components/Chips/Chips.tsx:138]: very cool!

File: src/components/Chips/Chips.tsx
Comment (hadasfa) [inline src/components/Chips/Chips.tsx:210]: cool !

**Review by hadasfa — APPROVED** [review]
Looking good!

File: src/components/Chips/Chips.tsx
Comment (SergeyRoyt) [reply src/components/Chips/Chips.tsx:115]: It's just always will equal empty string, even if ariaLabel and label are not passed - due to label default value.. maybe it's nothing, just don't like seeing this empty attributes in each snapshot. wdyt?

**Pull request merged**
